db/migrate/20161006152332_add_default_category_to_restaurants.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultCategoryToRestaurants < ActiveRecord::Migration[5.0]
+  def change
+    change_column_default :restaurants, :category, 'Other'
```

Comment:
I didn't add that migration.. so I don't know? Nobody's online so I can't ask.. should I delete it?